ludwig/schema/features/image_feature.py

```diff
@@ -15,7 +15,7 @@ class ImageInputFeatureConfig(BaseInputFeatureConfig):
 
     preprocessing: BasePreprocessingConfig = PreprocessingDataclassField(feature_type=IMAGE)
 
-    encoder: Optional[str] = schema_utils.StringOptions(
+    encoder: Optional[str] = schema_utils.StringOptionsOrURL(
```

Comment:
I believe this will also need to be refactored once encoder refactor lands since there is a nested structure now. Will probably be as simple as just moving this option to the corresponding parameter in the encoders, however this may have downstream effects in the rest of the PR.

Reply:
Good point.  I'm going to get rid of StringOptionsOrURL and just use a String field, similar to `pretrained_model: str = schema_utils.String(`